src/sqlfluff/dialects/dialect_ansi.py

```diff
@@ -1143,11 +1124,37 @@ class SetOperatorSegment(BaseSegment):
             )
         ),
         'INTERSECT',
-        'EXCEPT',
+        Sequence(
```

Comment:
This code is closely related to (i.e. it's the complement of) `ExceptSegment`. It might be good to add a comment mentioning the relationship (to make the code easier to navigate).

I wonder, would it also be a good idea to DRY up lines 1130-1135? In other words, since here we're saying `Not( ALL THE THINGS )`, it seems like it'd be safer to move those lines into their own segment and use it both here and in  `ExceptSegment`.